backend/templates/v1/signature.html

```diff
@@ -33,4 +33,15 @@
 
 {% block scripts %}
     <script type="text/javascript" src="{{ static('js/signature.js') }}"></script>
+    <script>
+        $(document).ready(function() {
+            $signature_send_btn = $("#signature-send")
+            setTimeout(function() {
+                $signature_send_btn.removeClass("hidden");
+            }, 2000);
```

Comment:
Can you only leave the `disabled` property?
It's weird not to see the button at all.
It may lead people who sign extra fast to click the only available button.